content/build/cloud/usage.md

```diff
@@ -0,0 +1,113 @@
+---
+title: Building with Cloud Build
+description: Invoke your cloud builds with the Buildx CLI client
+keywords: build, cloud build, usage, cli, buildx, client
+aliases:
+  - /hydrobuild/usage/
+---
+
+To build using Docker Cloud Build, invoke a build command and specify the name of the
+builder using the `--builder` flag.
+
+```console
+$ docker buildx build --builder cloud-<ORG>-default --tag <IMAGE> .
+```
+
+## Use by default
+
+If you want to use Docker Cloud Build without having to specify the `--builder` flag
+each time, you can set it as the default builder.
+
+{{< tabs group="ui" >}}
+{{< tab name="CLI" >}}
+
+Run the following command:
+
+```console
+$ docker buildx use cloud-<ORG>-default --global
+```
+
+{{< /tab >}}
+{{< tab name="Docker Desktop" >}}
+
+1. Open the Docker Desktop settings and navigate to the **Builders** tab.
+2. Find the cloud builder under **Available builders**.
+3. Open the drop-down menu and select **Use**.
+
+   ![Selecting the cloud builder as default using the Docker Desktop GUI](/build/images/set-default-builder-gui.webp)
+
+{{< /tab >}}
+{{< /tabs >}}
+
+Changing your default builder with `docker buildx use` only changes the default
+builder for the `docker buildx build` command. The `docker build` command still
```

Comment:
I don't think this is true on DD, as buildx is installed by default. This is still the case on github runners though.

Reply:
no, it's still true that `docker build` will not selected the "used" builder, without the `--builder` flag.

Reply:
Hmm I just tried using Docker Desktop:
```sh
$ docker buildx use my-cloud-builder
$ docker build . -t distribution
[+] Building 0.0s (0/0)                    cloud:my-cloud-builder
...
```